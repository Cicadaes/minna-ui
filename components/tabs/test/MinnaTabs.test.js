'use strict';

const MinnaTabs = require('../src/MinnaTabs.html');

describe('MinnaTabs component', () => {
  it('renders correctly with defaults', () => {
    const target = document.createElement('div');
    new MinnaTabs({ target });
    expect(target.innerHTML).toMatchSnapshot();
  });
});
