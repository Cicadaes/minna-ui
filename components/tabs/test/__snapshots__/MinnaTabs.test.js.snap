// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MinnaTabs component renders correctly with defaults 1`] = `""`;
